modules/statics/src/coins.ts

```diff
@@ -7834,9 +7834,9 @@ export const coins = CoinMap.fromCoins([
   erc20(
     '243053c9-e331-48f8-9f99-7af4e4f080c6',
     'uco',
-    'UnirisToken',
+    'Archethic Universal Coin',
     18,
```

Comment:
The decimal has been updated from 18 to 8, so this needs to be updated here.
<img width="1686" alt="Screenshot 2025-01-15 at 10 29 40 AM" src="https://github.com/user-attachments/assets/c5ca9b60-a691-4171-b2a4-c8177a847866" />